src/js/src/helper.cpp

```diff
@@ -42,12 +42,37 @@ ov::Any js_to_cpp<ov::Any>(const Napi::Env& env, const Napi::Value& value) {
             return ov::Any(num.DoubleValue());
         }
     } else if (value.IsBoolean()) {
-        return ov::Any(value.ToBoolean());
+        return ov::Any(static_cast<bool>(value.ToBoolean()));
     } else {
         OPENVINO_THROW("Cannot convert to ov::Any");
     }
 }
 
+template <>
+ov::genai::StringInputs js_to_cpp<ov::genai::StringInputs>(const Napi::Env& env, const Napi::Value& value) {
+    const auto elem = value;
```

Comment:
If the intention is to rename the var, you can rename the function arg. If the intention is to copy, you can remove reference from `const Napi::Value&` so it copies for you.

Reply:
It's just an artifact of debugging. Thank you for your attentiveness.